runtimes/huggingface/mlserver_huggingface/codecs.py

```diff
@@ -1,89 +1,351 @@
-from typing import Optional, Type, Any, Dict
+import json
+from typing import Optional, Type, Any, Dict, List, Union
+from functools import partial
+from xmlrpc.client import Boolean
```

Comment:
What is `xmlrpc`? Shouldn't this be `typing.Boolean` instead?

Reply:
Sorry, it's a typo, VSCODE auto add this and I forget to check.